### PR TITLE
Update pysam to 0.11

### PR DESCRIPTION
**Pull request by kyleabeauchamp**
* [ ] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

**This is a pre-release test, do not merge!**

### CODE REVIEW

**Comment by kyleabeauchamp** [comment]
cc @AndreasHeger  looks like your HTS1.4 branch is building the bioconda packages successfully AFAIK, so we just need to update this PR when the updates are complete.

File: recipes/pysam/meta.yaml [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
-    version: 0.10.0
+    version: 0.11.0
 
 source:
-    fn: pysam-0.10.0.tar.gz
-    url: https://pypi.python.org/packages/87/8e/d2d8238558970df37c7aa01ddec63057a98042334e939b4c1c69cb9a2504/pysam-0.10.0.tar.gz
-    md5: a1f3333ce60f8de542624cb07c792fc0
-    patches:  # [osx]
+  git_url: https://github.com/pysam-developers/pysam.git
+  git_rev: 697c9918338a638d251831d05d503af24c1773d3
```
Comment (bgruening) [inline recipes/pysam/meta.yaml]: @kyleabeauchamp you could also use https://github.com/pysam-developers/pysam/archive/697c9918338a638d251831d05d503af24c1773d3.tar.gz and don't have the problems with the linter.

Comment (bgruening) [inline recipes/pysam/meta.yaml]: assuming there is no git-submodule, but I haven't seen any.

Comment (kyleabeauchamp) [inline recipes/pysam/meta.yaml]: Good point, but this will switch to a tar as soon as the release happens.

**Comment by kyleabeauchamp** [comment]
ping @AndreasHeger  updated to point to pypi tarball.

**Pull request merged**